### PR TITLE
[Logs 1 of 2] DCOS-12769: Make paging in log view a bit snappier

### PR DESCRIPTION
**Pull request by mlunoe**
Logs 1 of 2 PR: https://github.com/dcos/dcos-ui/pull/1736
Logs 2 of 2 PR: https://github.com/dcos/dcos-ui/pull/1745

This PR also resolves: DCOS-12737

This PR fixes a few issues related to the same thing - making the paging become smoother:
- It initializes the logs no matter what response comes from the server, so we do not see 'Empty Log' when there is an error.
- It stop debouncing `componentDidUpdate` in  the log view, so it actually goes to bottom initially
- It removes log window so the position calculation in `componentDidUpdate` continues to work
- It lets the container handle when to retrieve a previous page, instead of assuming an ongoing request when at the top.

Initially, the paging goes pretty fast, but eventually the journald bug manifests and loading becomes increasingly slower. This is not handled by this PR.

### CODE REVIEW

**Comment by mlunoe** [comment]
Run unit tests

**Comment by mlunoe** [comment]
Run unit tests

File: plugins/services/src/js/components/LogView.js [hunk 1]
```diff
@@ -28,18 +28,12 @@ class LogView extends React.Component {
...
 
     this.handleWindowResize = Util.debounce(
       this.handleWindowResize.bind(this), 100
     );
-
-    // Since we can receive a lot of updates (from a stream potentially),
-    // let's debounce this function to not call it too often
-    this.componentDidUpdate = Util.debounce(
-      this.componentDidUpdate.bind(this), 100
-    );
```
Comment (jfurrow) [inline plugins/services/src/js/components/LogView.js]: Are we no longer concerned about receiving a lot of updates at once?

**Comment by jfurrow** [comment]
Works well!

File: plugins/services/src/js/components/LogView.js (hunk 1 above)
Comment (mlunoe) [inline plugins/services/src/js/components/LogView.js]: I am working on a performance PR that will take care of this. More or less I am using `shouldComponentUpdate` a little more actively

Comment (jfurrow) [inline plugins/services/src/js/components/LogView.js]: Okay cool, thanks for the clarification!

Comment (mlunoe) [inline plugins/services/src/js/components/LogView.js]: Removed the commit from this PR

**Pull request merged**